docs/docs/Configuration/configuration-objects.md

```diff
@@ -1,34 +1,24 @@
 ---
-title: Data & Message
-sidebar_position: 2
-slug: /guides-data-message
+title: Langflow Objects
+sidebar_position: 1
+slug: /configuration-objects
 ---
 
```

Comment:
Are there any other langflow-specific objects users might want to know about here?

Reply:
I don't think that we currently have it. The Memory type is going to be released soon, but I think the most important ones are Message and Data. @ogabrielluiz what do you think?

Reply:
Agree. Now we do have a DataFrame type that should be documented, though.

Reply:
I added content and examples for DataFrame and ContentObject (we had a question about it today)